CirculaireICTKeten/CirculaireICTKeten/Views/Shared/_Layout.cshtml

```diff
@@ -1,61 +1,70 @@
 ﻿<!DOCTYPE html>
+@using Microsoft.AspNetCore.Http;
 <html lang="en">
 <head>
-	<meta charset="utf-8" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-	<title>@ViewData["Title"] - CirculaireICTKeten</title>
-	<!--<link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />-->
-	<!-- CSS only -->
-	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
```

Comment:
Wij hadden bootstrap en fontawsome online ingeladen in plaats van lokaal. Willen we dit weer lokaal doen? want dat gebeurt met deze change

Reply:
Zoals via app besproken hebben beveiligingsproblemen minste prioriteit voor PoC. Daarnaast werkte dit in onze pipeline